new/rfcXXX-event-handler-dsl.md

```diff
@@ -0,0 +1,81 @@
+---
+RFC: unassigned
+Author: Ranjib Dey (ranjib@linux.com)
+Status: Draft
+Type: Standards Track
+---
+
+
+# Recipe DSL method for event handler hooks
+
+Allow cookbook authors easily add custom logic on Chef events.
+
+
+## Motivation
+
+Chef has an extensive event handler mechanism, but incorporating
+some custom logic with any of the events is onerous process which involves
+subclassing the based evant handler and adding it via the config. This RFC
+proposes a recipe DSL method to ease this. For new chef users this increases
```

Comment:
Might want to rearrange these sentences. When I first read this it seemed like it was saying the recipe DSL method increases the entry barrier, etc.